<ui version="4.0" >
 <comment> * OpenRPT report writer and rendering engine
 * Copyright (C) 2001-2012 by OpenMFG, LLC
 *
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 2.1 of the License, or (at your option) any later version.
 *
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this library; if not, write to the Free Software
 * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 * Please contact info@openmfg.com with any questions on this license.</comment>
 <class>ReportParameter</class>
 <widget class="QDialog" name="ReportParameter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameter</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="_lblName" >
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>_leName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="_lblType" >
         <property name="text" >
          <string>Type:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>_cbType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="_leName" />
       </item>
       <item row="1" column="1" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="_cbType" >
           <item>
            <property name="text" >
             <string>String</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Integer</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Double</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Bool</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_active" >
           <property name="text" >
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="_leDefault" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="_lblDefault" >
         <property name="text" >
          <string>Default:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>_leDefault</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="_btnOk" >
         <property name="text" >
          <string>&amp;OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btnCancel" >
         <property name="text" >
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="_tab" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="description" >
      <attribute name="title" >
       <string>Description</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="_tbDescrip" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lists" >
      <attribute name="title" >
       <string>Defined List</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="_listType" >
         <property name="title" >
          <string>List Type</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>5</number>
          </property>
          <property name="margin" >
           <number>5</number>
          </property>
          <item>
           <widget class="QRadioButton" name="_staticList" >
            <property name="text" >
             <string>Static</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="_dynamicList" >
            <property name="text" >
             <string>Dynamic</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="_stack" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="WStackPage" >
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="_lblStatic" >
             <property name="text" >
              <string>Static List Values:</string>
             </property>
             <property name="buddy" >
              <cstring>_listValues</cstring>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="margin" >
              <number>0</number>
             </property>
             <item>
              <widget class="QTreeWidget" name="_listValues" >
               <column>
                <property name="text" >
                 <string>Value</string>
                </property>
               </column>
               <column>
                <property name="text" >
                 <string>Label</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" >
               <property name="spacing" >
                <number>6</number>
               </property>
               <property name="margin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="_add" >
                 <property name="text" >
                  <string>Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="_edit" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="text" >
                  <string>Edit</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType" >
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>20</width>
                   <height>21</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="_remove" >
                 <property name="enabled" >
                  <bool>false</bool>
                 </property>
                 <property name="text" >
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dynamic" >
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>0</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="_lblDynamic" >
             <property name="text" >
              <string>Dynamic List Query:</string>
             </property>
             <property name="buddy" >
              <cstring>_dynamicQuery</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="_dynamicQuery" />
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>_leName</tabstop>
  <tabstop>_cbType</tabstop>
  <tabstop>_active</tabstop>
  <tabstop>_leDefault</tabstop>
  <tabstop>_tbDescrip</tabstop>
  <tabstop>_tab</tabstop>
  <tabstop>_staticList</tabstop>
  <tabstop>_listValues</tabstop>
  <tabstop>_add</tabstop>
  <tabstop>_edit</tabstop>
  <tabstop>_remove</tabstop>
  <tabstop>_dynamicList</tabstop>
  <tabstop>_dynamicQuery</tabstop>
  <tabstop>_btnOk</tabstop>
  <tabstop>_btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_btnOk</sender>
   <signal>clicked()</signal>
   <receiver>ReportParameter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>384</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ReportParameter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>384</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
